test/spec/controllers/tasksCtrlSpec.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 describe('Tasks Controller', function() {
-  var $rootScope, shared, scope, user, ctrl;
+  var $rootScope, shared, scope, user, User, ctrl;
 
   beforeEach(function() {
-    user = specHelper.newUser();
```

Comment:
Ah, yes. Sorry. I interpreted the other way because of the change I made. I have updated the code with this syntax.